rpc/client.go

```diff
@@ -257,6 +257,7 @@ func (c *rpcClient) SendRequest(ctx context.Context, addr string, req *Request,
 	// Coprocessor streaming request.
 	// Use context to support timeout for grpc streaming client.
 	ctx1, cancel := context.WithCancel(ctx)
+	defer cancel()
```

Comment:
Great catch, seem TiDB has the same problem now, please send a PR too
@rleungx

Reply:
Ok.